elastalert/kibana_external_url_formatter.py

```diff
@@ -157,3 +157,4 @@ def create_kibana_external_url_formatter(
         return ShortKibanaExternalUrlFormatter(base_url, auth, security_tenant, new_shortener, verify)
 
     return AbsoluteKibanaExternalUrlFormatter(base_url, security_tenant)
+
```

Comment:
A blank line has been added at the end, so please delete it.

Reply:
done